### PR TITLE
Inasafe 4289

### PR DESCRIPTION
**Pull request by timlinux**
### What does it fix?
* Ticket: #4289
* Funded by: WB/GFDRR
* Description: Prevents divide by zero error when we are calculating pie charts....

### CODE REVIEW

**Comment by timlinux** [comment]
Note this needs to be back ported to develop too...

File: metadata.txt [hunk 1]
```diff
@@ -35,7 +37,7 @@ homepage=http://www.inasafe.org/
 tracker=https://github.com/inasafe/inasafe/issues
 repository=https://github.com/inasafe/inasafe
 icon=resources/img/icons/icon.png
-# experimental flag
+# experimenal flag
```
Comment (Gustry) [inline metadata.txt]: What are you doing here? ;-)
Can you update the version number at the same time ?

File: safe/plugin.py [hunk 2]
```diff
@@ -930,7 +930,7 @@ def layer_changed(self, layer):
                         enable_field_mapping_tool = True
             else:
                 enable_field_mapping_tool = False
-        except (KeywordNotFoundError, NoKeywordsFoundError):
+        except (KeywordNotFoundError, NoKeywordsFoundError, MetadataReadError):
```
Comment (Gustry) [inline safe/plugin.py]: Did you put this fix in another PR already ?

**Comment by Gustry** [comment]
Closing this one.
